ui/components/AutomationDetail.tsx

```diff
@@ -87,10 +91,15 @@ function AutomationDetail({
       component: () => {
         return (
           <RequestStateHandler loading={isLoading} error={error}>
-            <ReconciledObjectsTable
-              className={className}
-              objects={data?.objects}
-            />
+            {automation.type === "HelmRelease" &&
+            (automation as HelmRelease).kubeConfig === "" ? (
```

Comment:
Seems that this change caused that only HelmRelease type can have detail now as reported in [this bug](https://github.com/weaveworks/weave-gitops/issues/4057).